app/controllers/menus_controller.rb

```diff
@@ -0,0 +1,8 @@
+class MenusController < ApplicationController
+
+  def show
+    @restaurant = Restaurant.find(params[:restaurant_id])
+    @menu = Menu.find_by(restaurant: @restaurant)
```

Comment:
For now a restaurant can only have one take out menu. It can´t have one lunch menu and one dinner menu. We decided in the group not to have that as an option.